src/main/default/lwc/subjectInput/subjectInput.js

```diff
@@ -0,0 +1,123 @@
+import { LightningElement, api } from 'lwc';
+import writeDescriptiveTitle from '@salesforce/label/c.cookbook_SubjectInputTitle';
+import errorValueMissing from '@salesforce/label/c.cookbook_ValueMissing';
+
+/**
+ * The `SubjectInput` component  displays a text input for the case subject.
+ * @example
+ * <c-subject-input label="Write a descriptive title" maxLength="50" message-when-value-missing="Complete this field." required></c-subject-input>
+ */
+export default class SubjectInput extends LightningElement {
+  labels = {
+    writeDescriptiveTitle,
+    errorValueMissing
+  };
+  /**
+   * The label of the input.
+   * @api
```

Comment:
Just a reminder that these `@api` jsdoc tags might not be useful to you as it's a custom tag we added that's only used by quantic's custom jsdoc template. Feel free to keep them if you prefer though.

Reply:
I will update this thanks👌